website/content/en/status/report-2023-04-2023-06/capsicum-ktracing.adoc

```diff
@@ -0,0 +1,230 @@
+=== Security Sandboxing Using `ktrace(1)`
+
+Links: +
+link:https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace[ktrace branch] URL: link:https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace[] +
+
+Contact: Jake Freeland <jfree@FreeBSD.org>
+
+==== Capsicumization With `ktrace(1)`
+
+This report introduces an extension to `ktrace(1)` that logs `ECAPMODE` Capsicum violations.
+
+The first logical step in Capsicumization is determining where your program is raising capability violations.
+You could approach this issue by looking through the source and removing Capsicum-incompatible code, but this can be tedious and requires the developer to be familiar with everything that is not allowed in capability mode.
+
+An alternative to finding violations manually is to use `ktrace`.
```

Comment:
You might want to switch all occurences of `` `ktrace` `` to `man:ktrace[1]`: it is not required, but since it generates a link to ktrace manpage without displaying its manpage section number, it might be nice.
It would also provide consistency in formatting.